### PR TITLE
Enable debounce at 100% on all channels

### PR DESCRIPTION
**Pull request by pilgrim-brave**

### CODE REVIEW

File: seed/seed.json
Comment (iefremov) [inline seed/seed.json:971]: this is a `HangWatcher` feature

Comment (pilgrim-brave) [reply seed/seed.json]: Sigh. Fixed.

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-variations/pull/195#issuecomment-1014872639.

**Comment by stephendonner** [comment]
Verified `PASSED` using 

Brave | 1.34.81 Chromium: 97.0.4692.99 (Official Build) (x86_64)
-- | --
Revision | d740da257583289dbebd2eb37e8668928fac5ead-refs/branch-heads/4692@{#1461}
OS | macOS Version 11.6.1 (Build 20G224)

Followed the verification steps from https://github.com/brave/brave-variations/pull/195#issuecomment-1014872639.

`Enabled`

Launched Brave using `--enable-logging=stderr --v=1 --variations-server-url=https://variations.bravesoftware.com/seed` and verified the following:

* ensured that `Brave Local Data Updater` > `Brave Local Data Updater - Version: 1.0.45` which was the version that wasn't working when initially running through the above. Most recent version appears to be `Brave Local Data Updater - Version: 1.0.52` which has the latest debounce list
* ensured that `BraveDebounceStudy:Enabled` under `brave://version` (after restarting twice due to a Griffin limitation)

example | example
---------|---------
Screen Shot 2022-01-20 at 2 04 32 PM | Screen Shot 2022-01-20 at 2 05 13 PM



Went through the cases under `https://dev-pages.brave.software/navigation-tracking/debouncing.html` and ensured the following:

`YT`

* ensured that clicking on `Run` returned to the test page and `Observed Referrer --> dev-pages.brave.software`

```
[7296:259:0120/140556.291001:VERBOSE1:debounce_throttle.cc(51)] SiteForCookies: {site=https://youtube.com; schemefully_same=true}
[7296:259:0120/140556.291095:VERBOSE1:debounce_throttle.cc(60)] Debouncing rule applied: https://www.youtube.com/redirect?q=https%3A%2F%2Fdev-pages.brave.software%2Fnavigation-tracking%2Fdebouncing.html%3Fcomplete -> https://dev-pages.brave.software/navigation-tracking/debouncing.html?complete
[7296:259:0120/140556.291165:VERBOSE1:debounce_throttle.cc(76)] SiteForCookies: {site=https://brave.software; schemefully_same=true}
```

`Reddit`

 * ensured that clicking on `Run` returned to the test page and `Observed Referrer --> dev-pages.brave.software`

```
[7296:259:0120/140620.161242:VERBOSE1:debounce_throttle.cc(51)] SiteForCookies: {site=https://reddit.com; schemefully_same=true}
[7296:259:0120/140620.161318:VERBOSE1:debounce_throttle.cc(60)] Debouncing rule applied: https://out.reddit.com/test?url=https%3A%2F%2Fdev-pages.brave.software%2Fnavigation-tracking%2Fdebouncing.html%3Fcomplete -> https://dev-pages.brave.software/navigation-tracking/debouncing.html?complete
[7296:259:0120/140620.161365:VERBOSE1:debounce_throttle.cc(76)] SiteForCookies: {site=https://brave.software; schemefully_same=true}
```

`Disabled`

Launched Brave using `--enable-logging=stderr --v=1 --variations-server-url=https://variations.bravesoftware.com/seed` and ensured the following:

`YT`

 * ensured that clicking on `Run` didn't return to the test page and `Observed Referrer --> https://www.youtube.com` when clicking on GO TO SITE
 * went through the logs and ensured that Debouncing rule applied wasn't visible as no rules are being applied

`Reddit`

 * ensured that clicking on Run didn't return to the test page (no debouncing occurring)
 * went through the logs and ensured that Debouncing rule applied wasn't visible as no rules are being applied

**Pull request merged**
